Unit3/Model/Config/Source/Dropdown.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RLTSquare\Unit3\Model\Config\Source;
+
+use Magento\Framework\Data\OptionSourceInterface;
+
+class Dropdown implements OptionSourceInterface
+{
+    /**
+     * @return array
+     */
+    public function toOptionArray(): array
+    {
+        return [
+            ['value' => 'Staging', 'label' => __('Staging')],
```

Comment:
You should declare value keys here as constants, e.g. "const ENV_STAGING = 'staging'", so you can later use RLTSquare\UnitThree\Model\Config\Source\Select::ENV_STAGING in if statements and other comparisons you need.